packages/components/avatar/src/avatar.ts

```diff
@@ -140,6 +147,13 @@ export class Avatar extends LitElement {
     this.borderWidth = this.#config?.badgeGapWidth * 2;
     this.setAttribute('shape', this.#config.shape);
 
+    this.#resizeObserver = new ResizeObserver(() => {
```

Comment:
This way a new observer will be created every time `connectedCallback` is called. That can be multiple times. It's probably better to create the observer directly when you declare the variable.

Reply:
I copied this from the grid, there it is also done on the connected callback, maybe it needs to change there as well?